packages/components/src/Api/Entitlements.purs

```diff
@@ -0,0 +1,25 @@
+module KSF.Api.Entitlements where
+
+import Prelude
+import Data.JSDate (JSDate)
+import Data.Maybe (Maybe(..))
+import Data.Nullable (Nullable)
+import Foreign (Foreign, ForeignError(..), fail)
+import Foreign.Object (lookup)
+import Simple.JSON (class ReadForeign, readImpl)
```

Comment:
Check imports, there are extra since you removed the `ReadForeign` instance.

Reply:
Should I get compiler warnings from unused imports? I don't.

Reply:
I found the unused imports and removed them from this module.